src/kanata/mod.rs

```diff
@@ -556,6 +559,13 @@ impl Kanata {
                 } else {
                     mmsv.distance = mmas.max_distance;
                 }
+                for &modifier in &self.move_mouse_speed_modifiers {
```

Comment:
To save on performance, this computation should only happen when `ticks_until_move` is 0